addons/web/static/src/legacy/js/core/minimal_dom.js

```diff
@@ -3,7 +3,7 @@
 import { addLoadingEffect } from '@web/core/utils/ui';
 
 export const DEBOUNCE = 400;
-export const BUTTON_HANDLER_SELECTOR = 'a, button, input[type="submit"], input[type="button"], .btn';
+export const BUTTON_HANDLER_SELECTOR = 'a, button, input[type="submit"], input[type="button"], .btn, input[type="radio"]';
```

Comment:
@alessandro-caldonazzi I probably was not clear yesterday: we probably don't want this. Indeed:

- First, it is strange as a "button handler selector": you see in this file that we call `addLoadingEffect` on the elements we find with this selector, that callbacks **adds** elements inside the target element... which we can't do on "radio" inputs.

- Then, mainly, "radio" inputs are things with a value (checked or unchecked), if the user clicks on it while it is not lazy loaded, it will check it (and doing nothing since no handler attached yet)... then replaying the click on it won't trigger the change event again. Does this actually fix the test issue? :thinking:  It might, if devs are listening to "click" or stranger things but I would not rely on it.

That replaying events system is something we want to limit anyway. It's a best effort feature, so not something that tests should rely on: I would implement the "wait for lazy loading" we discussed in the problematic tests.

Reply:
@qsm-odoo I tried that, but I was not able to wait for scripts inside the iframe. I guess the simplest way is just to avoid being in the iframe (doing a logout). When there is no iframe I saw that the tours automatically await for `allScriptsLoaded`. I made the change, what do you think?

Reply:
Mmh, feels a bit like hiding the issue but why not, we would still be testing what's originally wanted and it's a tour issue we'd be hiding, not a real one.

However, let's do that even better: instead of a logout in the middle of the tour, split the tour in two. And start the second part from the python, without a set user:
```python
self.start_tour("/?enable_editor=1", "donation_snippet_edition", login='admin')
self.start_tour("/?enable_editor=1", "donation_snippet_use")
```